admindoc_fr.md

```diff
@@ -3,23 +3,22 @@
 * [L’auto-hébergement](/selfhosting_fr)
 * [Matériel compatible](/hardware_fr)
 * [Guide d’installation](/install_fr)
-   * [Sur carte ARM](/install_on_arm_board_fr)
-   * [Sur Raspberry Pi](/install_on_raspberry_fr)
-   * [À partir de l’ISO](/install_iso_fr)
-   * [Sur Debian](/install_on_debian_fr)
-   * [Sur VirtualBox](/install_on_virtualbox_fr)
-   * [Sur Docker](/docker_fr)
-   * [Sur Vagrant](/vagrant_fr)
-   * [Sur un conteneur DigitalOcean](/install_on_digitalocean_fr)
-   * [Sur un serveur dédié](/install_on_dedicated_server_fr)
-   * Étapes d’installation
-      * [Graver ou copier l’ISO](/burn_or_copy_iso_fr)
-      * [Démarrage et installation graphique](/boot_and_graphical_install_fr)
-      * [Copier l’image YunoHost sur une carte SD](/copy_image_fr)
-      * [Installation manuelle](/install_manually_fr)
-      * [Brancher et démarrer votre serveur](/plug_and_boot_fr)
-      * [Post-installation](/postinstall_fr)
-      * [Diagnostic du bon fonctionnement du YunoHost](/diagnostic_fr)
+    * [Sur la Brique Internet](/installation_brique_fr)
```

Comment:
I really wonder if we should just redirect people to install.labriqueinter.net instead.